go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	cosmossdk.io/log v1.2.1
 	cosmossdk.io/math v1.1.2
 	cosmossdk.io/store v1.0.0-rc.0
+	cosmossdk.io/x/pool v0.0.0-20230907131022-067909526905
```

Comment:
Why the SDK needs to import pool?

Reply:
because we are using pool module in x/gov